Subject: Fix crosscompilation
From: Marc Kleine-Budde <mkl@pengutronix.de>

This patch fixes cross compilation, of another handcrafted
Makefile.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
#20081125: wsa: Mailed to upstream.
#20081127: wsa: Accepted upstream. Wait for next release.

 Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: atop-1.23/Makefile
===================================================================
--- atop-1.23.orig/Makefile
+++ atop-1.23/Makefile
@@ -13,8 +13,8 @@ INIPATH = /etc/rc.d/init.d
 CRNPATH = /etc/cron.d
 ROTPATH = /etc/logrotate.d
 
-CFLAGS  = -O -I. -Wall	 # -DHTTPSTATS
-LDFLAGS = -lncurses -lm -lz
+CFLAGS  += -O2 -I. -Wall	 # -DHTTPSTATS
+LDFLAGS += -lncurses -lm -lz
 OBJMOD0 = version.o
 OBJMOD1 = various.o  deviate.o   procdbase.o
 OBJMOD2 = acctproc.o photoproc.o photosyst.o  rawlog.o ifprop.o parseable.o
@@ -25,7 +25,7 @@ ALLMODS = $(OBJMOD0) $(OBJMOD1) $(OBJMOD
 all: 		atop
 
 atop:		atop.o    $(ALLMODS) Makefile
-		cc atop.o $(ALLMODS) -o atop $(LDFLAGS)
+		$(CC) atop.o $(ALLMODS) -o atop $(LDFLAGS)
 
 clean:
 		rm -f *.o
